packages/core/src/Stores/client-store.js

```diff
@@ -1584,6 +1584,29 @@ export default class ClientStore extends BaseStore {
         this.registerReactions();
         this.setIsLoggingIn(false);
         this.setInitialized(true);
+
+        const unusedParams = [
```

Comment:
- Where did you get the list from?
- I also suggest declare the variable at the top of the component.
- rename to `unused_params` (We only use camelCase naming conventions for functions)

Reply:
I got the list from the description of the card, where were the sample of URL

Reply:
I'll also rename all const and remove declaration to the top